src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -103,7 +107,15 @@ public Command parseCommand(String userInput) throws ParseException {
             }
 
         case ListCommand.COMMAND_WORD:
-            return new ListCommand();
+            if (arguments.trim().startsWith("-s")) {
```

Comment:
Will it be an issue if user types "list -sXXXXXX" as an input?

Reply:
I understand this issue I was gonna discuss with yall regarding the error handling for this